shared/services/config/stader-config.go

```diff
@@ -72,6 +73,7 @@ type StaderConfig struct {
 	StaderDirectory string `yaml:"-"`
 
 	IsNativeMode bool `yaml:"-"`
+	IsSSVMode    bool `yaml:"-"`
```

Comment:
Why using `-`, you should put it in `StaderNodeConfig`

Reply:
`stader-config` or `stader-node-config` is not big things you can choose the place. However, please refrain to use `-`, you will need to manually serialize it.

Reply:
setting a name will lead to handle default cases of yaml files that are not setting this for backward compatibility.
keeping it to de-serialize from the user settings where user is expected to manually add this to use ssv-mode for now.